docs/UsersGuide/source/Docker.rst

```diff
@@ -0,0 +1,607 @@
+.. _Docker:
+
+*************
+Run in Docker
+*************
+
+This document explains how to run the SRW public release, version 1,
+in a Docker container. We will run twice: once by building a Docker
+image complete with the application, and a second time building the
+application manually within a pre-existing Docker image.
+
+1. Get Docker
+2. Download the Files - get the premade container and input data
+3. Create the Docker Image - compile inside a docker image based on the premade container
+4. Start the Workflow - run the workflow off of the image you created
+5. Monitor the Workflow - look at log files while the workflow runs
+6. Where is my Output?
+7. Changing the Code - manually compile and run
+
+.. note::
+   If you find :ref:`DockerSection3` too hard, then
+   manually compile and run instead. It may be easier for you. To do
+   that, you would read the sections in this order instead:
+
+   * :ref:`DockerSection1`
+
+   * :ref:`DockerSection2`
+
+   * :ref:`DockerSection7`
+
+   * :ref:`DockerSection4`
+
+   * :ref:`DockerSection5`
+
+   * :ref:`DockerSection6`
+
+.. _DockerSection1:
+   
+Section 1: Get Docker
```

Comment:
The "Section 1:" is duplicating what is done with the rst section numbering.  It it showing up as:
"11.1. Section 1: Get Docker". I think we can delete the "Section 1:", and just use "Get Docker".  Can you fix this for all sections?

Reply:
@JulieSchramm Please check the latest changes and let me know if they're what you wanted.